### PR TITLE
RPJ Workflow

### PR DESCRIPTION
**Pull request by gabriel-farache**
Closes [FLPATH-1587](https://issues.redhat.com/browse/FLPATH-1587)
RPJ Workflow

### CODE REVIEW

File: rpj/rpj.sw.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      - name: createNotification
+        functionRef:
+          refName: createNotification
+          arguments:
+            recipients:
+              type: "entity"
+              entityRef: .recipients
+            payload:
+              title: "RPJ run results"
+              description: '"RPJ result for launch " + .launchId + ": Message: " + .runResults.message ' #+ "; Data: " + .runResults.data
```
[the remark this answers is not in the document]
Comment (gabriel-farache) [reply rpj/rpj.sw.yaml:111]: `data` is too big so I get a 413 error but I will eventually want to have it once/if they fix that, see https://backstage-backstage-rhdh-operator.apps.edge122-cluster-0.qe.lab.redhat.com/orchestrator/instances/f3a57b6f-9c3d-455e-9788-1e1f7bfb9ded results

File: rpj/specs/rpj.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          "message": {
+            "type": "string"
+          },
+          "data": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "x-original-swagger-version": "2.0"
```
[the remark this answers is not in the document]
Comment (gabriel-farache) [reply rpj/specs/rpj.json:96]: no idea, I used something to convert that to openapi spec, must be leftover

File: rpj/rpj.sw.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ElaiShalevRH) [reply rpj/rpj.sw.yaml:111]: 1. Is it too large even without the  [DEBUG] flag?
2. The results are supposed to be for the users, is this log helpful for them?

File: rpj/specs/rpj.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (gabriel-farache) [reply rpj/specs/rpj.json]: I removed it

File: rpj/rpj.sw.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gabriel-farache) [reply rpj/rpj.sw.yaml:111]: the `data` is in the sysout

[the remark this answers is not in the document]
Comment (ElaiShalevRH) [reply rpj/rpj.sw.yaml:111]: OK, makes sense to not display anything then, just maybe we could make the comment more informative

**Review by masayag — APPROVED** [review]

**Pull request merged**